reflex-dom-core/src/Reflex/Dom/Widget/Input.hs

```diff
@@ -437,12 +437,13 @@ dropdown k0 options (DropdownConfig setK attrs) = do
         let xs = fmap (\(ix, (k, v)) -> ((ix, k), ((ix, k), v))) $ zip [0::Int ..] $ Map.toList os
         in (Map.fromList $ map snd xs, Bimap.fromList $ map fst xs)
   modifyAttrs <- dynamicAttributesToModifyAttributes attrs
-  initialIxKeys <- sample $ current ixKeys
-  let initialValue = maybe "" (T.pack . show) $ Bimap.lookupR k0 initialIxKeys
+  postBuild <- getPostBuild
+  let updatedSelection = attachPromptlyDynWithMaybe (flip Bimap.lookupR) ixKeys setK
+      initialSelection = attachPromptlyDynWithMaybe (flip Bimap.lookupR) ixKeys (k0 <$ postBuild)
```

Comment:
Have you confirmed that promptness is necessary here? If so maybe add comment explaining why.